app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -23,11 +23,14 @@ public function __construct($record)
         $this->first_name = $record['First name'];
         $this->last_name = $record['Last name'];
         $this->mobile = $record['Phone'];
-        $this->source = config('services.northstar.client_credentials.client_id');
+        $this->source = get_user_source();
```

Comment:
We should be able to omit this field & Northstar will automatically set the source based on the OAuth client!

Reply:
Oh, I didn't know that! I'll remove this helper then.